src/main/java/org/gridsuite/study/server/dto/LoadFlowParametersInfos.java

```diff
@@ -0,0 +1,26 @@
+/**
+  Copyright (c) 2023, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto;
+
+import com.powsybl.loadflow.LoadFlowParameters;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import java.util.Map;
+
+/**
+ * @author David Braquart <david.braquart@rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@Builder
+public class LoadFlowParametersInfos {
```

Comment:
LoadFlowParametersInfos is sent to loadflow-server when a LF computation is started.
Same with SecurityAnalysisParametersInfos (new DTO) and SensitivityAnalysisInputData (existing DTO).
Each DTO contains "Map<String, String> specificParameters", which is used  by each computation server
to create speficic params extensions.
We pass only String values because the "value" column in the load_flow_specific_parameters table is not typed.

LoadFlowParametersValues is the DTO sent to the Front on the Get operation /studies/{studyUuid}/loadflow/parameters.
Because this call is made only a single time when we open the Study (existing design, cf hook useParametersBackend),
we return all parameters, for all LF providers.